frontend/src/navbar.tsx

```diff
@@ -9,7 +9,7 @@ import { faSnowflake, faMoon } from '@fortawesome/free-solid-svg-icons'
 export function Navbar(props: BoxProps) {
    const pulls: Pull[] = usePulls();
    const setPullFilter = useSetFilter();
-   const [darkmode, setDarkmode] = useBoolUrlState('dark', false);
+   const {toggleColorMode} = useColorMode();
```

Comment:
Look at the Chakra docs it seems they are using a state variable or the usage of `useColorModeValue`, but I am assuming since there is no state variable declared here that Chakra keeps track of the toggled color state?

Reply:
> Add colorModeManager (Optional, for SSR)

> If you don't use SSR or don't care about this, you don't need to pass anything. Chakra will use localStorageManager by default.

Docs say it's optional and the default uses local storage to keep the state. I concur as that's how it seems to work.